docs/servo_power_monitor/src/index.ts

```diff
@@ -94,22 +94,47 @@ function kickWriteLoop(writeFn: (s: string) => Promise<void>) {
 }
 async function readLoop(readFn: () => Promise<string>) {
   while (!halt) {
-    const s = await readFn();
-    if (s === undefined || !s.length) {
-      continue;
+    try {
+      const s = await readFn();
+      if (s === undefined || !s.length) {
+        continue;
+      }
+      pushOutput(s);
+    } catch (e) {
+      break;
```

Comment:
Adding a comment to explain why we can just ignore the error here will be useful for future developers ;)

Reply:
Thank you for your advice, I added comments!